Jenkinsfile

```diff
@@ -154,13 +154,37 @@ pipeline {
                 }
             }
             options {
-                timeout(time: 30, unit: 'MINUTES')
+                timeout(time: 40, unit: 'MINUTES')
             }
             steps {
                 script {
+                    def curr_params = createRunConfiguration('docker')
+                    def builder = getJenkinsLabels(curr_params.backend, curr_params.region, curr_params.gce_datacenter, curr_params.azure_region_name)
                     try {
-                        sh './docker/env/hydra.sh integration-tests'
-                        pullRequestSetResult('success', 'jenkins/integration-tests', 'All integration tests are passed')
+                        withEnv(["SCT_TEST_ID=${UUID.randomUUID().toString()}",]) {
+                            dir('scylla-cluster-tests') {
+                                checkout scm
+
+                                wrap([$class: 'BuildUser']) {
+                                    echo "calling createSctRunner"
+                                    timeout(time: 5, unit: 'MINUTES') {
+                                        createSctRunner(curr_params, 50 , builder.region)
+                                    }
+                                }
+                                sh """#!/bin/bash
+                                    set -xe
+                                    echo "start integration-tests ..."
+                                    RUNNER_IP=\$(cat sct_runner_ip||echo "")
+                                    if [[ -n "\${RUNNER_IP}" ]] ; then
+                                        ./docker/env/hydra.sh --execute-on-runner \${RUNNER_IP} integration-tests
+                                    else
+                                        ./docker/env/hydra.sh integration-tests
```

Comment:
we would actually never going to get here, if creation of runner would fail.

yeah this part can be remove, it's just me copying the logic from other steps

Reply:
removed the if